packages/web-runtime/src/components/SidebarNav/SidebarNav.vue

```diff
@@ -141,6 +141,22 @@ export default {
   .toggle-sidebar-button-expanded {
     justify-content: flex-end !important;
   }
+
+  .oc-sidebar-nav li a {
+    &:hover,
+    &:focus {
+      text-decoration: none !important;
+      background-color: var(--oc-color-background-hover);
+      color: var(--oc-color-swatch-passive-default);
```

Comment:
can you reproduce the following? in dark mode there is a slight delay until the text is rendered in the brighter color. This is not observable in light mode because there the text color is the same as without highlighting.

Reply:
Indeed. I've removed the transition (see https://github.com/owncloud/web/pull/7567/commits/8f1d848dcf797ee22b8fd15ecf134871224efaf4), please check again. I didn't notice any weird behavior after removing this.

Reply:
👍